compute_tools/src/metrics.rs

```diff
@@ -59,10 +61,20 @@ pub(crate) static REMOTE_EXT_REQUESTS_TOTAL: Lazy<IntCounterVec> = Lazy::new(||
     .expect("failed to define a metric")
 });
 
+// Size of audit log directory in bytes
+pub(crate) static AUDIT_LOG_DIR_SIZE: Lazy<GenericGauge<AtomicF64>> = Lazy::new(|| {
+    register_gauge!(
+        "compute_ctl_audit_log_dir_size",
```

Comment:
NIT: can be just
```suggestion
        "compute_audit_log_dir_size",
```

as it's a property of compute, not just compute_ctl